packages/ui-babel-preset/lib/index.js

```diff
@@ -176,7 +176,8 @@ function getWebEnvConfig(opts) {
       browsers: require('@instructure/browserslist-config-instui')
     },
     useBuiltIns: 'entry',
-    corejs: 3,
+    // this version has to match the version in package.json
+    corejs: '3.26.1',
```

Comment:
According to the [babel docs](https://babeljs.io/docs/en/babel-preset-env#corejs) this has to be a string.